modules/apps/site/site-insurance-site-initializer/src/main/zippableResources/fragments/raylife/fragments/get-in-touch/index.css

```diff
@@ -11,12 +11,10 @@ ul {
 	padding: 0px;
 }
 
-.btn-back { 
+.content-agent-btn { 
```

Comment:
Just to maintaining the consistence, at line 26 and 31, you used **btn-** before, and here -btn **after**

It could be **btn-content-agent**

Reply:
Also, this class should come after:

.btn-back-color
.btn-call-color

We need to put this class, and properties ordered by name (ASC)